packages/terra-arrange/docs/Arrange.md

```diff
@@ -0,0 +1,9 @@
+| Prop Name | Type | Is Required | Default Value | Description |
```

Comment:
I am not opposed to providing the prop markdown in the documentation. I think it could be useful to consumers, though I would vote for a consistent naming scheme like `PROPS.md` that could be referenced from the `README.md` file.